src/mono/browser/runtime/es6/dotnet.es6.lib.js

```diff
@@ -88,12 +87,13 @@ function injectDependencies() {
         `linkerEnableAotProfiler: ${ENABLE_AOT_PROFILER ? "true" : "false"}, ` +
         `linkerEnableBrowserProfiler: ${ENABLE_BROWSER_PROFILER ? "true" : "false"}, ` +
         `linkerRunAOTCompilation: ${RUN_AOT_COMPILATION ? "true" : "false"}, ` +
-        `gitHash: "${gitHash}", ` +
+        `linkerUseThreads: ${USE_PTHREADS ? "true" : "false"}, ` +
```

Comment:
Has this the same meaning as `WasmEnableThreads`? 
If so, could we unify the name with others (eg `linkerWasmEnableSIMD`)?

Reply:
this is at dev machine, during re-link. It should match `WasmEnableThreads` but sometimes it doesn't because of stale browser cache. This is baked into `dotnet.native.js` and it makes sense to compare expectations.